index.html

```diff
@@ -0,0 +1,330 @@
+              <li>javascript</li>
+            </ul>
+            <a href="#">See Project</a>
+          </div>
+        </div>
+      </section>
+      <!-- section 2 ends Here -->
+
+      <!-- section 3 starts here -->
+      <section class="section3">
+        <div class="about">
+          <h1>About Myself</h1>
+          <p>
+            Hello I’m a software developer! I can help you build a product,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+
+          <div class="social"><p>LET’S CONNECT</p></div>
+
+          <div class="icons">
+            <ul>
+              <li>
+                <a href="#"><i class="fa fa-facebook fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-twitter fa-2x"></i></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-instagram fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-github fa-2x"></i></a>
+              </li>
+
+              <li>
+                <a href="#"><i class="fa fa-linkedin fa-2x"></i></a>
+              </li>
+            </ul>
+          </div>
+          <div class="get-resume">
+            <button type="button" class="get-resume-btn">Get My Resume</button>
+          </div>
+        </div>
+
+        <div class="accordion">
+          <ul class="mother-accordion">
+            <li class="accordion-item">
+              <h2>
+                <span>Languages</span>
+                <i class="fa fa-angle-down" aria-hidden="true"></i>
+              </h2>
+              <ul class="child-accordion">
+                <li>
+                  <div class="language-name">
+                    <img src="./image/language/js.svg" alt="" />
+                    <span>JavaSript</span>
+                  </div>
+                </li>
+                <li>
+                  <div class="language-name">
+                    <img src="./image/language/html.svg" alt="" />
+                    <span>HTML</span>
+                  </div>
+                </li>
+                <li>
+                  <div class="language-name">
+                    <img src="./image/language/css.svg" alt="" />
+                    <span>CSS</span>
+                  </div>
+                </li>
+              </ul>
+            </li>
+            <li class="accordion-item">
+              <h2>
+                <span>Frameworks</span>
+                <i class="fa fa-angle-right" aria-hidden="true"></i>
+              </h2>
+            </li>
+            <li class="accordion-item">
+              <h2>
+                <span>Skills</span>
+                <i class="fa fa-angle-right" aria-hidden="true"></i>
+              </h2>
+            </li>
+          </ul>
+        </div>
+      </section>
```

Comment:
- [ ] kindly fix the alignments of this section, try to reduce some margins and font sizes to match Figma design


Your design          | Figma
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/193111734-3ebc9eba-69c0-4f8a-a3a5-27cc231324c8.png) | ![image](https://user-images.githubusercontent.com/89282221/193111402-23f17fb5-9e12-4e48-914e-8444474e0423.png)